bundle/lib/source/DobbyTemplate.cpp

```diff
@@ -361,72 +358,6 @@ void DobbyTemplate::setTemplateEnvVars(const std::map<std::string, std::string>&
     ctemplate::TemplateDictionary::SetGlobalValue("EXTRA_ENV_VARS", envVarsStream.str());
 }
 
-// -----------------------------------------------------------------------------
-/**
- *  @brief Sets the environment variables used to identify the platform.
- *
- *  All containers get two environment variables that define the platform;
- *
- *      ETHAN_STB_TYPE  = [ "GW" | "MR" | "HIP" ]
- *      ETHAN_STB_MODEL = [ "ES140" | "ES130" | "EM150" | "ES240" | "ESi240" | ... ]
- *
- *  For a list of model names refer to the following confluence page
- *
- *      https://www.stb.bskyb.com/confluence/display/2016/STB+HW+ID
- *
- *  To determine platform we check for the AI_PLATFORM_IDENT environment
- *  variable.  This should have been set by the user (or upstart script) that
- *  started this daemon.  If it's not present then we set both templates
- *  as empty and log an error.
- */
-void DobbyTemplate::setTemplatePlatformEnvVars()
```

Comment:
Disagree about the need to remove this method - will need to remain for full compatibility with Sky builds